src/map/lua/lua_baseentity.cpp

```diff
@@ -4906,86 +4874,87 @@ inline int32 CLuaBaseEntity::getModelId(lua_State* L)
  *  Notes   :
  ************************************************************************/
 
-inline int32 CLuaBaseEntity::setModelId(lua_State* L)
+void CLuaBaseEntity::setModelId(uint16 modelId, uint8 slot = 0)
```

Comment:
When we get around to testing this stuff, I _think_ in order to use default arguments in place of lua nil, we have to use `sol::object` as the argument, and then do a round of checks against it.

Gross I know, but it's what we're left with as we move away from stack-based argument grabbing